specs/pac/pac-gitlab.spec

```diff
@@ -0,0 +1,17 @@
+PIPELINES-29
+# Pipelines As Code tests
+
+## Enable/Disable PAC: PIPELINES-29-TC01
+Tags: pac, sanity, e2e
+Component: PAC
+Level: Integration
+Type: Functional
+Importance: Critical
+
+This scenario tests enable/disable of pipelines as code from tektonconfig custom resource
```

Comment:
We usually don't use this kind of comments. I don't mind it but at least update `enable/disable` part and fix uppercase in PAC and TektonConfig